spec/mongo/auth/aws/request_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe Mongo::Auth::Aws::Request do 
+
+  describe "#formatted_time" do 
+    original_time = Time.now.freeze
+    let(:request) do 
+      described_class.new(access_key_id: 'access_key_id', 
+        secret_access_key: 'secret_access_key', 
+        session_token: 'session_token', 
+        host: 'host', 
+        server_nonce: 'server_nonce',
+        time: original_time
+      )
+    end
+
+    it 'doesn\'t modify the time instance variable' do
+      expect { request.formatted_time }.to_not raise_error
+    end
```

Comment:
I would love to see an example that tests the behavior of the method -- what should formatted_time return? Does it actually do that? I would also love to see the description of the test match what is being tested.